pandas/core/groupby.py

```diff
@@ -2315,6 +2315,11 @@ def __init__(self, index, grouper=None, obj=None, name=None, level=None,
                 # groupby
                 else:
                     cat = self.grouper.unique()
```

Comment:
This [_is_ vectorized](https://github.com/pandas-dev/pandas/pull/15439/commits/0c550e6b70e17d10c8f296578ce24a033e1283bc#diff-720d374f1a709d0075a1f0a02445cd65R2319), right?